examples/todoapp/README.md

```diff
@@ -9,26 +9,31 @@ Libraries used:
 - [Reaktive](https://github.com/badoo/Reaktive) - background processing and data transformation
 - [SQLDelight](https://github.com/cashapp/sqldelight) - data storage
 
-There are multiple common modules:
-- `utils` - just some useful helpers
-- `database` - SQLDelight database definition
-- `main` - displays a list of todo items and a text field
-- `edit` - accepts an item id and allows editing
-- `root` - navigates between `main` and `edit` screens
+There are multiple modules:
+- `:common:utils` - just some useful helpers
+- `:common:database` - SQLDelight database definition
+- `:common:main` - displays a list of todo items and a text field
+- `:common:edit` - accepts an item id and allows editing
+- `:common:root` - navigates between `main` and `edit` screens
+- `:common:compose-ui` - Shared Compose UI for `main` and `edit` screens
+- `:android` - Android application
+- `:desktop` - Desktop application
+- `ios` - iOS Xcode project
 
-The `root` module is integrated into both Android and Desktop apps.
+The `root` module is integrated into Android, Desktop and iOS apps.
 
 Features:
 - 99% of the code is shared: data, business logic, presentation, navigation and UI
 - View state is preserved when navigating between screens, Android configuration change, etc.
 - Model-View-Intent (aka MVI) architectural pattern
+- Pluggable UI - Compose UI for Android and Desktop, SwiftUI for iOS
```

Comment:
Pluggable UI - Compose UI for Android and Desktop, SwiftUI (not Compose) for iOS